yairl/util/util.py

```diff
@@ -228,32 +179,34 @@ def load_policy(env, policy_class=stable_baselines.PPO2, basedir="",
       constructor. Unless we are using a custom BasePolicy (not builtin to
       stable_baselines), this is automatically infered, and so we can leave
       this argument as None.
-    verbose (int): The verbosity level of the model during training.
     **kwargs: Additional options for initializing the BaseRLModel class.
     """
-    path = os.path.join(basedir, _policy_filename(policy_class, env))
-    exists = os.path.exists(path)
-    if exists:
-        env = maybe_load_env(env)
-        if policy_network_class is not None:
-            kwargs["policy"] = policy_network_class
-
-        policy = policy_class.load(path, env,
-                tensorboard_log=_get_tb_log_dir(env, init_tensorboard),
-                **kwargs)
-        logging.info("loaded policy from '{}'".format(path))
-        return policy
-    else:
-        logging.info("couldn't find policy at '{}'".format(path))
-        return None
 
+    paths = get_policy_paths(env, policy_model_class, basedir, n_experts)
+
+    env = maybe_load_env(env)
+
+    if policy_network_class is not None:
+        kwargs["policy"] = policy_network_class
```

Comment:
What if the user specifies `policy` in `kwargs`? Are we OK overwriting this? Perhaps rename `policy_network_class` to just `policy?`

Reply:
Fixed